flake.nix

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
Delete the `flake` and `.envrc` files for now, NixOS isnt a dev priority to support and it crowds the repo